frontend/src/features/idir/search/components/IDIRSearchApplicationLists.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { TabLayout } from "../../../../common/components/dashboard/TabLayout";
+import { IDIRApplicationSearchResults } from "./IDIRApplicationSearchResults";
+import { SearchFields } from "../types/types";
+import { StartApplicationAction } from "../../../permits/pages/TermOversize/components/dashboard/StartApplicationAction";
+
+export const IDIRSearchApplicationLists = React.memo(
+    ({
+        searchParams,
+      }: {
+        /**
+         * The search parameters entered by the user.
+         */
+        searchParams: SearchFields;
+      }) => {
+
+  const tabs = [
```

Comment:
Why are you building out a page for applications for staff? A staff user looks at applications in progress inside the company. Can you link the specs your work is based on?